modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,77 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM POWER8 or POWER9 processor-based systems
+
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance IBM POWER8 or POWER9 processor-based systems
+
+On your IBM Power instance, set up:
+
+* 3 guest virtual machines for {product-title} control plane machines
+* 2 guest virtual machines for {product-title} compute machines
+* 2 guest virtual machine for the temporary {product-title} bootstrap machine
+
+[discrete]
+=== Disk storage for the Power guest virtual machines
+
+* FICON attached disk storage (DASDs). These can be Power minidisks, fullpack minidisks, or dedicated DASDs. To reach the minimum required DASD size for {op-system-first} installations, you need extended address volumes (EAV). If available, use HyperPAV to ensure optimal performance.
+* FCP attached disk storage
+
+[discrete]
+=== Storage / Main Memory
+
+* 16 GB for {product-title} control plane machines
+* 8 GB for {product-title} compute machines
+* 16 GB for the temporary {product-title} bootstrap machine
+
+[id="preferred-ibm-Power-system-requirements_{context}"]
+
+== Preferred IBM Power system requirements
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance IBM POWER8 or POWER9 processor-based systems.
+
+On your Power instance, set up:
+
+* 3 guest virtual machines for {product-title} control plane machines
+* 2 guest virtual machines for {product-title} compute machines
+* 2 guest virtual machine for the temporary {product-title} bootstrap machine
+
+[discrete]
+=== Disk storage for the Power guest virtual machines
+
+* FICON attached disk storage (DASDs). These can be Power minidisks, fullpack minidisks, or dedicated DASDs. To reach the minimum required DASD size for {op-system-first} installations, you need extended address volumes (EAV). If available, use HyperPAV and High Performance FICON (zHPF) to ensure optimal performance.
```

Comment:
FWIK, DASD is supported on z. Also this line refers to zHPF but the header says POWER. Can you please recheck the content here @ktania46

Reply:
Can you provide what is Power substitute for DASD, or who can provide information on this ? @arcprabh

Reply:
@manojnkumar - pls check my comment and suggest the right term if we can use DASD for power.